src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -531,6 +532,25 @@ export const encryptSubmission = async (
     publicKey,
   )
 
+  // Modify response data for webhook responses and encrypt separately
+  const strippedResponsesWebhook = strippedBodyResponses.map((response) => {
+    if (
+      response.fieldType === BasicField.Signature &&
+      response.answerArray.length > 0
+    )
+      return {
+        ...response,
+        answerArray: [SIGNATURE_CAPTURED_STRING],
```

Comment:
question: not sure, but just to confirm if `answerArray` is supposed to differ from `answer` used below.

Reply:
thanks for explaining this!